docs/concepts.md

```diff
@@ -136,3 +137,22 @@ There are lot of interesting things happening in this example, which bring out t
 
 The intention of merge on read storage, is to enable near real-time processing directly on top of Hadoop, as opposed to copying
 data out to specialized systems, which may not be able to handle the data volume.
+
+## Trade offs when choosing different storage types and views
+
+### Storage Types
+
+| Trade-off | CopyOnWrite | MergeOnRead |
+|-------------- |------------------| ------------------|
+| Data Latency | High   | Low |
+| Query Latency | Low (raw columnar performance) | High (merge columnar + row based delta) |
+| Update cost (I/O) | High (rewrite entire parquet) | Low (append to delta file) |
```

Comment:
can you make everything relative terms? "High" ->  "Higher" , "Low" -> "Lower" , "Small" -> "Smaller" .. Currently I get the impression that for e.g COW writes small parquet files.

Reply:
done